package/MDAnalysis/coordinates/chain.py

```diff
@@ -271,6 +271,13 @@ def __init__(self, filenames, skip=1, dt=None, continuous=False,
                         for filename in filenames]
         self.filenames = np.array([fn[0] if isinstance(fn, tuple) else fn
                                                         for fn in filenames])
+        # Set filenames array to array of "None" if chainreader initialized with
```

Comment:
Also, it seems to me that the if/else you've added wholly replaces lines 272-273 above :-)